src/main/java/com/iota/iri/controllers/AddressViewModel.java

```diff
@@ -38,6 +42,30 @@ private AddressViewModel(Address hashes, Indexable hash) {
         this.hash = hash;
     }
 
+    /**
+     * Constructor for an {@link Address} set controller from an existing {@link Address} set.
+     * The resulting hash list is sorted
+     *
+     * @param tangle The tangle reference for the database to find the {@link Address} set in
+     * @param hash The hash identifier for the {@link Address} set that needs to be found
+     * @return The {@link AddressViewModel} controller generated
+     * @throws Exception Thrown if the database cannot load an {@link Address} set from the reference {@link Hash}
+     */
+    public static List<TransactionViewModel> loadAsSortedList(Tangle tangle, Indexable hash) throws Exception {
```

Comment:
Just note (no need to change):
Usually it is best to use an interface like `Indexable` in method parameters.

In this case it would probably best to use something more specific like `AddressHash` because you are not going to be using other subtypes of `Indexable` in this method.
However at the moment casting will be neccessary to change this...

So this is just a point to think of before we start refactoring...